src/realm/CMakeLists.txt

```
@@ -352,7 +352,7 @@ endif()
 # On systems without a built-in SHA-2 implementation (or one provided by a dependency)
 # we need to bundle the public domain implementation.
 # Note: This is also used on Windows because Windows lacks a native SHA224 hash needed for realm encryption
-if(NOT APPLE AND NOT REALM_HAVE_OPENSSL OR WIN32)
+if(NOT APPLE AND NOT REALM_HAVE_OPENSSL AND WIN32)
```

```suggestion
if(NOT APPLE AND NOT REALM_HAVE_OPENSSL)
```
If we're not running on Apple platforms and we don't have OpenSSL we need the fallback sha2 implementation. This includes Linux without encryption, for example.